index.html

```diff
@@ -0,0 +1,97 @@
+    <input type="number" name="age" id="number" min="10" max="99" class="form-control" placeholder="Age">
+    </div>
+    <div class="form-group">
+    <p>Which option best describes your current role?</p>
+    <select id="dropdown" name="role" class="form-control" required="">
+    <option disabled="" selected="" value="">Select current role</option>
+    <option value="student">Student</option>
+    <option value="job">Full Time Job</option>
+    <option value="learner">Full Time Learner</option>
+    <option value="preferNo">Prefer not to say</option>
+    <option value="other">Other</option>
+    </select>
+    </div>
+    <div class="form-group">
+    <p>Would you recommend the ApolloPets App to a friend?</p>
+    <label>
+    <input name="user-recommend" value="definitely" type="radio" class="input-radio" checked="">Definitely</label>
+    <label>
+    <input name="user-recommend" value="maybe" type="radio" class="input-radio">Maybe</label>
+    <label><input name="user-recommend" value="not-sure" type="radio" class="input-radio">Not sure</label>
+    </div>
+    <div class="form-group">
+    <p>
+    What is your favorite feature of freeCodeCamp?
+    </p>
+    <select id="most-like" name="mostLike" class="form-control" required="">
+    <option disabled="" selected="" value="">Select an option</option>
+    <option value="challenges">Challenges</option>
+    <option value="projects">Projects</option>
+    <option value="community">Community</option>
+    <option value="openSource">Open Source</option>
+    </select>
+    </div>
+    <div class="form-group">
+    <p>
+    What would you like to see improved?
+    <span class="clue">(Check all that apply)</span>
+    </p>
+    <label><input name="prefer" value="front-end-projects" type="checkbox" class="input-checkbox">Front-end Projects</label>
+    <label>
+    <input name="prefer" value="back-end-projects" type="checkbox" class="input-checkbox">Back-end Projects</label>
+    <label><input name="prefer" value="data-visualization" type="checkbox" class="input-checkbox">Data Visualization</label>
+    <label><input name="prefer" value="challenges" type="checkbox" class="input-checkbox">Challenges</label>
+    <label><input name="prefer" value="open-source-community" type="checkbox" class="input-checkbox">Open Source Community</label>
+    <label><input name="prefer" value="gitter-help-rooms" type="checkbox" class="input-checkbox">Gitter help rooms</label>
+    <label><input name="prefer" value="videos" type="checkbox" class="input-checkbox">Videos</label>
+    <label><input name="prefer" value="city-meetups" type="checkbox" class="input-checkbox">City Meetups</label>
+    <label><input name="prefer" value="wiki" type="checkbox" class="input-checkbox">Wiki</label>
+    <label><input name="prefer" value="forum" type="checkbox" class="input-checkbox">Forum</label>
+    <label><input name="prefer" value="additional-courses" type="checkbox" class="input-checkbox">Additional Courses</label>
```

Comment:
- Use list element to make more attractive your code <ul> <li>

Reply:
- [OPTION ] Always develop good words and precise terms that describe more about what you want the student to modify based on the requirements on the given milestone; please improve on this next time. :wink:  :smile: